### PR TITLE
fix unstable tests in IOS

### PR DESCRIPTION
**Pull request by helen-dikareva**
/сс @AlexanderMoskovkin

### CODE REVIEW

File: src/client/runner/automation/playback/scroll.js [hunk 1]
```diff
@@ -35,13 +35,16 @@ export default class ScrollAutomation {
     }
 
     static _setScroll (element, { left, top }) {
+        var scrollLeft = Math.max(left, 0);
```
Comment (AlexanderMoskovkin) [inline src/client/runner/automation/playback/scroll.js]:
```js
left = Math.max(left, 0)
```

**Comment by AlexanderMoskovkin** [comment]
lgtm when the tests are ok

**Comment by AndreyBelym** [comment]
lgtm

**Comment by inikulin** [comment]
Seems like it doesn't help: https://travis-ci.org/DevExpress/testcafe/jobs/99439737 \cc @helen-dikareva

**Pull request merged**